guardowl/protocols.py

```diff
@@ -382,17 +159,12 @@ def _run_simulation(
             """
         )
 
-        system = parameters.system
-
-        qsim = SimulationFactory.create_simulation(
-            system,
-            parameters.testsystem.topology,
-            platform=parameters.platform,
-            temperature=temperature,
-            env=parameters.env,
-            device_index=parameters.device_index,
-            ensemble=ensemble,
-        )
+    @staticmethod
+    def _run_simulation(
+        parameters: StabilityTestParameters,
+        qsim: Simulation,
```

Comment:
What does the q represent in qsim as defined in the qsim variable used in this file?

Reply:
thanks for pointing that out. that was to distinquish between a ml simulation object and a classical one, but this is now obsolute. Will remove the q.